### DISCUSSION TITLE
Suggestion to form an "unstructured mesh generation working group"

### DISCUSSION THREAD

**Discussion by saeed-moghimi-noaa** [noaa-ocs-modeling/OCSMesh#139] [Ideas]
## Unstructured mesh generation working group

All, 

After a good amount of communications, seems like several teams at NOAA, US and internationally are working on the same problem of mesh generation for coastal ocean models. 

I would like to suggest a working group to sit together and decide about a path forward. 

**Please tag relevant folks to this conversation here** so we can find a time to discuss.

Thanks,
-Saeed

@SorooshMani-NOAA  @AliS-Noaa  @aliabdolali  @brey  @feiye-vims  @josephzhang8 @jasonfleming @krober10nd  @pmav99 @SiqiLiOcean @tomsail @WPringle @melissamattwig @afortunato66 @felicio93 @kiaghadi2013 **others!?**

**Comment by evalseth** [comment]
Eirik Valseth

**Comment by KyeongPark** [comment]
Kyeong Park (parkk@tamug.edu)

**Comment by henokdemissie1** [comment]
Henok Kefelegn
